### ISSUE TITLE
Add support for WITH NAME references

### ISSUE THREAD

**Issue by jimni** [AmailP/robot-plugin#17]
I am using plugin version 0.12 with Pycharm and would very much like to be able to reference libraries using custom names. At this moment plugin fails to recognize a keyword as correct for such library:

<img width="406" alt="2" src="https://cloud.githubusercontent.com/assets/15658803/12113352/286032b0-b3b3-11e5-990e-fc50ae78da5e.png">

Other than that the plugin is great! Thanks, @AmailP!

**Comment by AmailP** [comment]
hey,

I try to open the link that you have sent but i get not found.
Can you anyway describe a bit more in depth the feature that you are
requesting?

Thank you anyway for your support,

Valerio

**Comment by jimni** [comment]
Sure.

After I import a library with "Import Library WITH NAME" keyword:
```
Import Library  Selenium2Library  WITH NAME  S2L
```
I would like to be able to use autocomplete, proofing and other features of your plugin with this new alias:
```
S2L.Set Selenium Speed  1
S2L.Open Browser  http://google.com
(...)
```

I also left a screenshot in the issue on github https://github.com/AmailP/robot-plugin/issues/17

**Comment by MichaelAroustian** [comment]
I have this issue as well. I thought it was a bug and was about to report it. 
Any ideas when this feature may see the light of day?

**Comment by skhomuti** [comment]
Resolved in #84
